src/message.py

```diff
@@ -24,36 +25,53 @@ def __init__(self, code, body=None):
 
     def send_response(self):
         """Format reponse for in http protocol."""
-        message = '{0} {1}{2}{3}'.format(
+        # message = '{0} {1}{2}{3}'.format(
+        #     HTML_PROTOCOL,
+        #     RESPONSE_CODE[self.code],
+        #     CRLF + CRLF,
+        #     self.body)
+        message = '{0} {1}{2}'.format(
             HTML_PROTOCOL,
             RESPONSE_CODE[self.code],
-            CRLF + CRLF,
-            self.body)
-        return message.encode('utf8')
+            CRLF + CRLF)
+        message = message.encode('utf-8')
+        try:
+            message += self.body[1]
+        except TypeError:
+            if self.code == 200:
+                html_list = '<html>'
+                for item in self.body[1]:
+                    html_list += '<li>{0}'.format(item)
```

Comment:
This is not an html list, but an html tag with some li tags inside it, which has no end tag.  That's not the same thing.  Write valid html always.